docs/governance.rst

```diff
@@ -0,0 +1,253 @@
+  | `Ross Markello`_ (@rmarkello)     | 0.5h/week       |
+  +-----------------------------------+-----------------+
+  | `Taylor Salo`_ (@tsalo)           | 3h/week         |
+  +-----------------------------------+-----------------+
+  | `Joshua Teves`_ (@jbteves)        | 0.5h/week       |
+  +-----------------------------------+-----------------+
+  | `Eneko Uruñuela`_ (@eurunuela)    | 0.5h/week       |
+  +-----------------------------------+-----------------+
+  | `Kirstie Whitaker`_ (@KirstieJane)| 0.5h/week       |
+  +-----------------------------------+-----------------+
+
+
+Rules
+`````
+
+1. Potential modifications to the Repository should first be proposed via an 
+   Issue. Rules regarding Votes apply to both Pull Requests and Issues.
+
+   - Every modification of the specification (including a correction of a typo, adding a new Contributor, an extension adding support for a new data type, or others) or proposal to release a new version needs to be done via a Pull Request (PR) to the Repository.
+2. Anyone can open a PR (this action is not limited to Contributors).
+3. PRs adding new Contributors must also add their GitHub names to the 
+   `all-contributors file`_. 
+   This should be done with the allcontributors bot.
+
+   - Contributors may also add themselves to the Zenodo file if they wish, but this is not mandatory.
+4. A PR is eligible to be merged if and only if these conditions are met:
+
+   a) The PR features at least two `Reviews that Approve <https://help.github.com/articles/about-pull-request-reviews/#about-pull-request-reviews>`_
+      the PR from Maintainers of which neither is the author of the PR. 
+      The reviews need to be made after the last commit in the PR (equivalent to 
+      `Stale review dismissal <https://help.github.com/articles/enabling-required-reviews-for-pull-requests/>`_
+      option on GitHub).
+   b) Does not feature any `Reviews that Request changes <https://help.github.com/articles/about-required-reviews-for-pull-requests/>`_.
+   c) Does not feature "WIP" in the title (Work in Progress).
+   d) Passes all automated tests.
+   e) Is not proposing a new release or has been approved by at least one
+      Maintainer (i.e., PRs proposing new releases need to be approved by
+      at least one Maintainer).
+5. After consultation with contributors, the steering committee can decide
+   to merge any PR - even if it's not eligible to merge according to Rule 4.
+6. Any Maintainer can Review a PR and request changes. If a Maintainer Requests
+   changes they need to provide an explanation regarding what changes should
+   be added and justification of their importance. Reviews requesting
+   changes can also be used to request more time to review a PR.
+7. A Maintainer who Requested changes can Dismiss their own review or Approve
+   changes added by the Contributor who opened the PR.
+8. If the author of a PR and Maintainer who provided Review that Requests
```

Comment:
I'm not entirely sure I follow this rule; could you edit for clarity @handwerkerd ?

Reply:
I agree, I'm also a bit lost here.

Reply:
@tsalo write this and can hopefully explain more. I assume it means that a maintainer is allowed to change their mind if they asked for a change that they later decided wasn't necessary.

Reply:
This is one of the rules I took from BIDS. My understanding is that, if a maintainer is being intractable and requesting unreasonable changes, the rest of the maintainers can override their review based on a vote.